modules/codelite/codelite_project.lua

```diff
@@ -196,8 +196,8 @@
 		end
 
 		local toolset = m.getcompiler(cfg)
-		local cxxflags = table.concat(table.join(toolset.getcxxflags(cfg), cfg.buildoptions), ";")
-		local cflags   = table.concat(table.join(toolset.getcflags(cfg), cfg.buildoptions), ";")
+		local cxxflags = table.concat(table.join(toolset.getcxxflags(cfg), toolset.getforceincludes(cfg), cfg.buildoptions), ";")
+		local cflags   = table.concat(table.join(toolset.getcflags(cfg), toolset.getforceincludes(cfg), cfg.buildoptions), ";")
```

Comment:
Maybe a local var to avoid the double processing of `getforceincludes`?

Reply:
I did consider this, and for some reason decided to go with what I have. I'll change this though, makes way more sense to have a local.

Reply:
Fixed.